src/Strip.h

```diff
@@ -1,14 +1,18 @@
+#ifndef __STRIP_H__
+#define __STRIP_H__
 #include <NeoPixelBus.h>
 
 #define TEST_DELAY 15
+
+//TODO :: change this to not need float conversion
 #define REL_UNIT_BYTE 0.0039f
-#define REL_UNIT_SIZE 1.0f /  STRIP_SIZE
+
 // #define BENCHMARK
 #define MAX_LENGTH 300 //TODO:: temporal solution for dinamic array implementation
 
 #define CMD_FX "fx"
 #define CMD_OFF "off"
-#define CMD_SETHSL "setHsl"
```

Comment:
why rename this? `setHsl` is more descriptive.. `set` might be whatever.. no even a color.

Reply:
this is indeed strange, I didn't remove it myself, it's probably a bad rebase